### ISSUE TITLE
[iOS] convert to Objective-C

### ISSUE THREAD

**Issue by alamboley** [JackAppDev/flutter_iap#35]
Just read the #34 issue.

Swift is actually an issue for this plugin:
- it results in compilation issue with other plugins and manual changes to Podfile.
- it bloats the App size because it adds the Swift standard library. Kotlin will do the same.

Moving back to Objective-C will also enable to share code with strong plugin from other communities like [RN](https://github.com/dooboolab/react-native-iap).

**Comment by JackAppDev** [comment]
Good point. Swift has been causing issues yet I don't know Obj-C nor do I plan on learning it. If somebody would like to take up maintaining the ios side of it this would be a good change.

**Comment by thosakwe** [comment]
I've learned a decent bit of Objective-C recently, and would like to contribute to this project.

However, I *do* know way more Java, so if anything, I'll probably start with #34.

**Comment by ened** [comment]
Swift runtime size is a temporary issue which will be resolved once iOS ships with a swift standard library by default.

Cocoapods issues are 3rd party and as such should not be treated as showstoppers. This IAP (and other swift-based flutter plugins) work fine. Issues can usually be found with environment setup or actual configuration. Flutter officially supports both kotlin and swift and my suggestion is to follow mainstream flutter support lines here and report integration bugs there so it can be fixed in the future.
